tools/ops/azure/devops-flow/README.md

```diff
@@ -0,0 +1,111 @@
+# Cloud Custodian CI Pipelines
+
+This directory contains the yaml definitions for CI/CD pipelines to help manage policies in an Azure DevOps repository structure. The two pipelines will:
+* Validate policies as part of a Pull Request by running `custodian validate`
+* Copy the policy files into a blob storage container that will be picked up by a running container host. This pipeline takes a variable for controlling the folder that should be copied.
+
+In order to use these pipelines directly, you must use a repository in Azure DevOps, however the steps in both of the pipelines can easily be ported to the format of other build providers. If you do not have an existing project, [Create a Project in Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops) before continuing on.
+
+The other needed Azure resource is a storage account with a container to store the policy files. If you follow the directions for hosting Cloud Custodian in a container in Azure, this account will already be created. Otherwise please follow the instructions in `docs/source/azure/configuration/containerhosting.rst`. 
```

Comment:
Can we add a link to the published docs rather than rst file?

Reply:
Yes, I changed it to reference the Azure documentation. Should be the most up to date